pkg/cli/startbuild/startbuild.go

```diff
@@ -171,6 +173,7 @@ func NewCmdStartBuild(fullName string, f kcmdutil.Factory, streams genericcliopt
 	cmd.Flags().StringVar(&o.FromArchive, "from-archive", o.FromArchive, "An archive (tar, tar.gz, zip) to be extracted before the build and used as the binary input.")
 	cmd.Flags().StringVar(&o.FromRepo, "from-repo", o.FromRepo, "The path to a local source code repository to use as the binary input for a build.")
 	cmd.Flags().StringVar(&o.Commit, "commit", o.Commit, "Specify the source code commit identifier the build should use; requires a build based on a Git repository")
+	cmd.Flags().StringVarP(&o.ExcludeRegExp, "exclude", "", tar.DefaultExclusionPattern.String(), "When using the --from-dir option: regular expression for selecting files from the source tree to exclude from the build; the default excludes the '.git' directory (see https://golang.org/pkg/regexp for syntax, but note that \"\" will be interpreted as allow all files and exclude no files)")
```

Comment:
is this changing the default behavior? (was `.git` excluded by default before?)  we can't change default behaviors.

Reply:
https://github.com/openshift/oc/blob/master/pkg/helpers/source-to-image/tar/tar.go#L138

Reply:
Yes `.git` was excluded by default, per the default exclusion pattern in `tar.go`:

https://github.com/openshift/oc/blob/master/pkg/helpers/source-to-image/tar/tar.go#L29

Reply:
My use case was actually that I needed to *include* `.git` and thus needed a flag to control that pattern.

Reply:
ack, thanks